src/deluge/storage/flash_storage.cpp

```diff
@@ -286,11 +294,15 @@ void resetSettings() {
 	gridAllowGreenSelection = true;
 	defaultGridActiveMode = GridDefaultActiveModeSelection;
 
+
+
 	defaultMetronomeVolume = kMaxMenuMetronomeVolumeValue;
 
 	resetAutomationSettings();
 
 	defaultStartupSongMode = StartupSongMode::BLANK;
+
+	defaultSliceMode = SliceModedefaultOnce;
```

Comment:
you want to have this be = SampleRepeatMode::Once

defaultSliceMode is a variable, which will hold a SampleRepeatMode. SampleRepeatMode is a type which describes how samples repeat